$OpenBSD: patch-configure_ac,v 1.4 2016/10/12 11:56:53 sthen Exp $

Disable the noisy warnings, it makes it hard to see the more important basic
problems (missing prototypes for internal functions, etc - lots of sloppiness)
in amongst the chaff.

Prefer libedit over libreadline.

--- configure.ac.orig	Sat Oct  8 09:08:59 2016
+++ configure.ac	Wed Oct 12 12:46:02 2016
@@ -32,7 +32,7 @@ AC_CHECK_FUNCS([alarm gethostbyname getaddrinfo getifa
 AC_CHECK_FUNCS([memmove memset strchr strdup strerror])
 AC_CHECK_FUNCS([getpassphrase])
 
-CFLAGS="$CFLAGS -Wall -Wextra -std=gnu99 -pedantic -Wformat -Wformat-nonliteral"
+CFLAGS="$CFLAGS -std=gnu99 -Wformat"
 
 AM_PROG_LIBTOOL
 LIBTOOL="$LIBTOOL --silent"
@@ -570,7 +570,7 @@ AC_SUBST(IPMITOOL_INTF_LIB)
 if test "x$xenable_ipmishell" = "xyes"; then
 	AC_SEARCH_LIBS([tgetent], [tinfo ncurses curses readline termcap])
 	AC_SEARCH_LIBS([initscr], [ncurses curses], [have_curses=yes])
-	AC_SEARCH_LIBS([readline], [readline edit], [have_readline=yes])
+	AC_SEARCH_LIBS([readline], [edit readline], [have_readline=yes])
 	if test "x$have_curses" != "xyes" || test "x$have_readline" != "xyes"; then
 		xenable_ipmishell=no
 	fi
